### PR TITLE
Post warning when settings parsing fails in schedule timer

### PR DESCRIPTION
**Pull request by khandelwalbrijesh**

### CODE REVIEW

File: src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs [hunk 1]
```diff
@@ -344,7 +344,17 @@ private bool UpdateSettingsAndCreateCheckpoint(bool updateAttempted = true)
             {   // create temporary copy of settings.xml             
                 this.CreateTempCopyOfSettingsFile();
 
-                this._settingsManager.UpdateSettings(this._settingsManager.TempCopyofSettingsFilePath);
+                try
+                {
+                    this._settingsManager.UpdateSettings(this._settingsManager.TempCopyofSettingsFilePath);
+                    string message = "Attempt to update settings was successful.";
```
Comment (raunakpandya) [inline src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs]: >Attempt to update settings was successful. [](start = 38, length = 42)

Change to successfully updated the settings.

File: src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs [hunk 2]
```diff
@@ -344,7 +344,17 @@ private bool UpdateSettingsAndCreateCheckpoint(bool updateAttempted = true)
...
+                    this._nodeAgentSfUtility.ReportHealth(WUOperationSetting, message, HealthState.Ok, 1);
```
Comment (raunakpandya) [inline src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs]: >ReportHealth [](start = 45, length = 12)

You are posting this on the service. This kind of exception should be on the instance.

Comment (khandelwalbrijesh) [reply src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs]: The warning when LoadSettings in NodeAgentNTService is also posted on Service. Plus, we have used "WUOperationSetting"  to post these kind of events only.

Comment (khandelwalbrijesh) [reply src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs]: We are not posting anything on instance of NodeAgentService.

**Pull request merged**